Tosr/Common.cs

```
@@ -63,9 +65,9 @@ public enum Fase
 
     public static class Common
     {
-        private readonly static Dictionary<Fase, bool> FasesWithOffset = JsonConvert.DeserializeObject<Dictionary<Fase, bool>>(Resources.FasesWithOffset);
         public static bool HasOffset(this Fase fase)
         {
+            Dictionary<Fase, bool> FasesWithOffset = JsonConvert.DeserializeObject<Dictionary<Fase, bool>>(File.ReadAllText("FasesWithOffset.json"));
             FasesWithOffset.TryGetValue(fase, out bool result);

```

What happens if the fase is not there? Please throw an exception. Maybe keynotfoundexception?